### DISCUSSION TITLE
bug(Table): CardView 模式在特定情况下超出父组件

### DISCUSSION THREAD

**Discussion by AiYuZhen** [dotnetcore/BootstrapBlazor#4065] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

CardView 模式在特定情况下超出父组件

### Expected Behavior

超出后显示滚动条。

### Steps To Reproduce

[Uploading TableTest.zip…]()

### .NET Version

vs2022 17.10.5   .net 8.0

**Comment by bb-auto** [comment]
@AiYuZhen Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by AiYuZhen** [comment]
重现示例，第一次没有上传成功。
[TableTest.zip](https://github.com/user-attachments/files/16608802/TableTest.zip)

**Reply by ArgoZhang** [reply, marked as answer]
@AiYuZhen 这个我觉得不是 bug 是 `card-body` 少了样式

```
.card-body {
  height: 100%;
}
```

这是 Table 组件外围容器的问题，跟 Table 组件无关
